notes.md

```diff
@@ -358,6 +358,96 @@ it as 0x53B9 instead of 0x53B8. `Matroska Readers` may support these legacy file
 Matroska v2 or 0x53B9. The older values were 0: mono, 1: right eye, 2: left eye, 3: both eyes.
 
 
+# Default track selection
+
+This section provides some example sets of Tracks and hypothetical user settings, along with
+indications of which ones a similarly-configured `Matroska Player` **SHOULD** automatically
+select for playback by default in such a situation. A player **MAY** provide additional settings
+with more detailed controls for more nuanced scenarios. These examples are provided as guidelines
+to illustrate the intended usages of the various supported Track flags, and their expected behaviors.
+
+## Audio Selection
+
+Example track set:
```

Comment:
you should describe (in parenthesis), what each field is:
(track type, language, audio channels, original , default flags).

Or maybe a table would look better to differentiate flags.

Reply:
Agreed that this is clearer as a table; reformatted and re-pushed.